Dockerfile

```diff
@@ -1,20 +1,34 @@
 FROM node:6-alpine
 
 # install script dependencies
-RUN apk update && apk add sudo curl git wget
+RUN apk update && apk add \
+  bash \
+  curl \
+  g++ \
+  git \
+  make \
+  python \
+  sudo \
+  wget
+
+# install global npm dependencies
+RUN npm install -g ember-cli@latest && npm install -g bower
+
+# use changes to package.json to force Docker not to use the cache
+# when we change our application's nodejs dependencies:
+COPY package*.json /tmp/
+RUN cd /tmp && npm install
+RUN mkdir -p /usr/src/app && cp -a /tmp/node_modules /usr/src/app
+
+COPY bower.json /tmp/
```

Comment:
bower goes here

Reply:
you don't need this

Reply:
Why not? Saves several seconds time on rebuild. I remember that we're gonna get rid of bower but we haven't yet